apps/storybook/docs/a_afnemen/3_starter-app.stories.mdx

```diff
@@ -0,0 +1,149 @@
+te vragen.
+
+
+## Bibliotheken
+
+- [node](https://nodejs.org/en) - LTS versie - v20.13.1
+- [volta](https://volta.sh/) - installeer de laatste versie
+- [webpack](https://webpack.js.org/) - v5.91.0
+- [typescript](https://www.typescriptlang.org/) - v5.4.5
+- [Lit](https://lit.dev/) - v3.1.3
+- [prettier](https://prettier.io/) - v3.2.5
+- [ESLint](https://eslint.org/) - v8.57.0
+- [Jest](https://jestjs.io/) - v29.7.0
+- [Cypress](https://www.cypress.io/) - v13.10.0
+
+
+## Aanpak
+
+Concept:
+
+> Binnen Departement Omgeving streven we naar een Framework agnostische aanpak. Er wordt bewust niet ingezet op een
+concreet framework, dus geen React, Angular, Vue, ... . Het doel is toepassingen te bouwen op een native manier:
+met Html, Javascript en Css - gebruik makend van web-standaarden waarbij het bouwen van
+[Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) de basis vormt.
+
+
+Praktisch:
+
+Niettegenstaande de framework-agnostische aanpak willen we wel op een moderne manier aan frontend ontwikkeling doen -
+en vermijden het warm water uit te vinden. Een toepassing wordt gebouwd op het Node-platform en gebundeld met
+Webpack. Het is aanbevolen een linter te gebruiken en optioneel kan code in Typescript geschreven worden.
+Om kwalitatieve code te behouden is het de bedoeling 3 types van testen te schrijven: unit testen (met Jest) en
+component- en e2e-testen (met Cypress).
+
+In het verleden werden web-componenten van 'scratch' geschreven, hierdoor ontstond een basis-klasse die resulteerde
+in een mini-custom-framework. Om dat te vermijden is er Lit geïntroduceerd: een (minimale) open-source bibliotheek
+met Web Components als basis.
+
+
+## Opzet
+
+Om te starten met een nieuwe toepassing vertrek je van een fork van de starter-app. Deze maakt gebruik van Volta,
+dat is een tool die er voor zorgt dat er een expliciete Node versie gespecifiëerd is. Als je ontwikkeling doet voor
+meerdere toepassingen zal je daardoor telkens met de juiste Node versie werken.
+
+Je kan de nieuwste versie van Volta eenmalig als volgt installeren:
+```
+# install Volta
+curl https://get.volta.sh | bash
+
+# install Node
+volta install node
+
+# start using Node
+node
+```
+
+
+## Scripts
+
+In de package.json onder `scripts` staan verschillende shortcuts die je kan uitvoeren met `npm run {script}`.
+
+- **build**: build een bundel, je vind hem in de `dist` folder
+- **ct**: start lokaal de Cypress console om component testen uit te voeren
+- **ct-ci**: voert alle Cypress component testen uit
+- **e2e**: start lokaal de Cypress console om end-to-end testen uit te voeren
+- **e2e-ci**: voert alle Cypress end-to-end testen uit
+- **dev**: start lokaal een 'live' ontwikkel server
+- **lint**: voert de linter uit
+- **serve**: start een http-server waarmee de gebuilde bundel getest kan worden
+- **test**: voert alle Jest testen uit
+
+
+## Tooling
```

Comment:
Zeer goed dat je dit hebt neergeschreven, maar lijkt me dat we deze sectie nog ergens anders ook moeten vermelden. Als de afnemer nooit de pagina `starter-app` opendoet, zal die deze Tooling sectie ook niet tegenkomen.

Kan dit een aparte sectie worden en dat er dan verwezen wordt naar deze sectie vanuit deze documentatie starter-app documentatie pagina?

Dan kunnen wij zelf ook op andere plaatsen (waar relevant) verwijzen naar de Tooling (richtlijnen?) sectie.

Reply:
Zoals besproken, gaat wordt deze opmerking momenteel niet weerhouden wegens gebrek aan mandaat om standaarden af te dwingen.